main.nf

```diff
@@ -394,6 +396,9 @@ process markDuplicates {
         ASSUME_SORTED=true \\
         PROGRAM_RECORD_ID='null' \\
         VALIDATION_STRINGENCY=LENIENT
+
+    #Printing out version number to standard out
+    echo "File name: $bam_markduplicates MarkDuplicates version "$(java -Xmx2g -jar $PICARD_HOME/picard.jar  MarkDuplicates --version 2>&1)
```

Comment:
Yeah you are technically correct. I got misslead by the picard help info. Since you can't run `--version` on just the `Picard` jar. You need to specify a subprogram so I assumed that it displayed the verdion for that one, but no it's the same for all the subprograms. Changing it.